ofcourse-templates/calendar.html

```diff
@@ -0,0 +1,84 @@
+{% macro assignment_cell(values) %}
+  <td>
+    {% for _, name, link in values %}
+      <p class="topic ">
+      {% if link %}
+          <a target="_blank" href="{{link}}">{{name|safe}}</a>
+      {% else %}
+          {{name}}
+      {% endif %}
+      </p>
+    {% endfor %}
+  </td>
+{% endmacro %}
+
+{% macro title_cell(ev) %}
+  {% set cats = normalize_categories(ev) %}
+  <td><p class="topic {{cats|join(" ")}} {{"special" if "meetup" in cats else ''}}">
+    {% if 'attach' in ev %}
+        <a target="_blank" href="{{ev.decoded('attach')}}">
+            {{ev['summary']}}
+        </a>
+    {% else %}
+        {{ev['summary']}}
+    {% endif %}
+  </p></td>
+{% endmacro %}
+
+{% macro class_period(n, ev) %}
+  <td class="sessionnumber">{{n}}</td>
+  <td>{{ev.decoded('dtstart').strftime("%x")}}</td>
```

Comment:
I think you could put a more useful time here, for example `%a %h %d %H:%M` which yields "Mon Oct 10 15:00" since there's dispute between America and literally everyone else on the planed about MM/DD/YYYY vs DD/MM/YYYY.